flexmeasures/data/schemas/scheduling/__init__.py

```diff
@@ -31,3 +32,52 @@ class FlexContextSchema(Schema):
     inflexible_device_sensors = fields.List(
         SensorIdField(), data_key="inflexible-device-sensors"
     )
+
+
+class SequentialFlexModelSchema(Schema):
+    sensor = SensorIdField(required=True)
+    sensor_flex_model = fields.Dict(data_key="sensor-flex-model")
```

Comment:
Couldn't this wrap the single device flex model schema? Either extend it via inheritance or nesting it?

Reply:
It could, but I don't want to actually deserialize the sensor flex model here, because it needs to be passed to the job kwargs in serialized form. I agree that it's neater, though. I could see if I can do something with telling the endpoint decorator to partially deserialize the schema, i.e. only the sensor field. [At first glance it seems possible](https://marshmallow.readthedocs.io/en/stable/quickstart.html#partial-loading).

Reply:
We actually don't have a single device flex model schema. We currently have the `ProcessSchedulerFlexModelSchema` and the `StorageFlexModelSchema`, and potentially others in the future. Right now the use of `fields.Dict` already suggests that the `SequentialFlexModelSchema` is agnostic to the underlying types of flex models, so that has its merits, too.